.kitchen.yml

```diff
@@ -8,11 +8,13 @@ driver:
 
```

Comment:
Needed to add locally in order to run the test

customize:
    cableconnected1: 'on'